### DISCUSSION TITLE
这三个样式哪个好看些

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#516] [General]
人物头像边上的透明程度， chrome/chromium 76 开始支持的 `backdrop-filter` 滤镜
![image](https://user-images.githubusercontent.com/9006264/146960950-ca34c845-82a2-42f0-ab14-a88e4c89229e.png)
![image](https://user-images.githubusercontent.com/9006264/146960973-704af7b4-871d-4c3b-8832-96d80e0f5647.png)
![image](https://user-images.githubusercontent.com/9006264/146960986-0e395b4c-4d51-40ee-b191-e6fa1b5254e6.png)

**Comment by Arondight** [comment]
我觉得都差不多，没有什么特别的喜好

**Comment by mark9804** [comment]
火狐你支棱起来啊火狐，御三家又只剩你不支持了![image](https://user-images.githubusercontent.com/9006264/147030654-bea34b72-c5ff-4593-8363-7b344d5afd5d.png)
